workflow/README.md

```diff
@@ -0,0 +1,47 @@
+# TPS Pipeline
+
+**TPS workflow overview**
+
+![Alt Text](https://github.com/ajshedivy/tps/blob/visualization_v2/workflow/TPS_workflow.png)
+
+The TPS/workflow module utilizes the TPS repository structure in order to run the TPS software end-to-end in order to produce Cytoscape sessions. 
+
+## Documentation
+
+- `TPS.sh`
+  - Runs TPS software 
+  - sets up and initializes workspace environment `py2cyto_envirnment`
+  - calls driver script `main.py`
+
+- `main.py`
+  - redirects TPS output files
+  - calls utility functions for processing network annotations
+  - calls visualization utilities
+  
+- `visualization_utilities.py`
+  - utility functions used for creating cyREST client 
+  - import ooutput groah files, style and annotation into Cytoscape
+  - TODO : search machine for Cytoscape executable
+  
+- `table_refactor.py`
+  - helper methods of reformatting annotations data
+
+
+## Usage
+
+In order to run pipeline, call
+
+`bash TPS.sh` 
+
+from workflow directory
+
+***Note**: within `main.py`, change this [line](https://github.com/ajshedivy/tps/blob/visualization_v2/workflow/main.py#L62) to the install directory of Cytoscape
```

Comment:
We should change this to be a command line argument or environment variable.  We don't want users to have to modify the code.

An environment variable like `CYTOSCAPE_PATH` could be read by Python using the `os` module.

There is also a extra `*` in this line.